assets/css/input.scss

```diff
@@ -0,0 +1,204 @@
+$coral: #dee4ff;
+$indigo: #767EE9;
+$darkcoral: #acbbfb;
+$pink: #e5c5f3;
+$black: #2d2929;
+
+* {
+  box-sizing: border-box;
+}
+
+img {
+  width: 100%;
+}
+
+body {
+  font-family: 'Montserrat', sans-serif;
+  background: #f9f9fa;
+  color: $black;
+}
+
+#header {
+  text-align: center;
+  padding: 1em;
+
+  h1 {
+    font-size: 36px;
+    margin: 0;
+    background: $coral;
+    display: inline-block;
+    padding: 0.25em 0.5em;
+    border-radius: 5px;
+    border: 2px solid $indigo;
+    box-shadow: 8px 8px $indigo;
+    color: $black;
+  }
+}
+
+h1, h2 {
+  font-family: 'Inconsolata', monospace;
+  letter-spacing: 0.05em;
+  line-height: 1.25;
+}
+
+.text-image-container {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0 1.5em;
+  margin: 3em auto;
+  max-width: 1200px;
+  position: relative;
+
+  >div {
+    flex-basis: 50%;
+  }
+
+  .ti-text {
+    background: $coral;
+    border: 2px solid $indigo;
+    border-radius: 5px;
+    padding: 2.5em 1.5em;
+    box-shadow: 8px 8px $indigo;
+
+    p, a {
+      margin-top: 0;
+      line-height: 1.5;
+      letter-spacing: 0.04em;
+    }
+
+    a {
+      position: relative;
+      color: $indigo;
+      z-index: 2;
+      text-decoration: none;
+      font-weight: bold;
+      letter-spacing: 0.06em;
+      padding: 0 0.25em;
+
+      &:after {
+        content: '';
+        left: -0.01em;
+        right: -0.01em;
+        bottom: 0;
+        top: 65%;
+        background: $pink;
+        position: absolute;
+        z-index: -1;
+        transition: top 0.25s ease;
+      }
+
+      &:hover, &:focus {
+        outline: none;
+        &:after {
+          top: 0;
+        }
+      }
+    }
+  }
+
+  .ti-img {
+    position: relative;
+
+    .ti-img-bg {
+      position: absolute;
+      top: 2em;
+      bottom: -2em;
+      left: -2em;
+      right: 0;
+      background: $darkcoral url("/assets/images/computer.png") repeat-x center;
```

Comment:
Any chance we can get this as an SVG? Just looks nicer and is lighter in weight 🙂

Reply:
Ah only available as an SVG annoyingly :/